client/src/components/application/Examine/ActionButtons.vue

```diff
@@ -291,8 +291,8 @@ export default {
 
         // initialize user in edit mode, with previous state set so NR gets set back to draft
         //  when user is done changing name, adding comment, etc.
-        this.$store.state.previousStateCd = 'DRAFT';
```

Comment:
just noticed some sloppy coding here mutating state directly, changed to mutation